doc/notebooks/jupyter_integration.ipynb

```diff
@@ -53,7 +53,8 @@
    "outputs": [],
    "source": [
     "# Let's check our import by printing docstring for init()\n",
-    "gj.init?"
+    "gj.init?\n",
+    "gj.grassRenderer?"
```

Comment:
Only the last line seems to really "counts" here, so you need decide or do two different ones. I would say only one is needed or push the second one later in the nb.